cli/types/tests/plugins-config.ts

```diff
@@ -16,6 +16,8 @@ const pluginConfig2: Cypress.PluginConfig = (on, config) => {
   config.version // $ExpectType: string
   config.testingType // $ExpectType: TestingType
 
+  config.browsers // $ExpectType: false
```

Comment:
@lmiller1990 this is what I was most recently looking into. This was set as a failing test but it doesn't seem to be triggered. Looking to see where we regressed in terms of running the test on these